ansible/roles/prosody/files/mod_muc_auth_vpaas.lua

```diff
@@ -120,16 +145,17 @@ local function validate_vpaas_token(session)
         -- first uuid is the customer id
         -- a customer can change the public key so the last uuid after /
         -- uniquely identifies the key for that customer
-        local _, tenant_uuid_from_kid, _ = kid:match("^(vpaas%-magic%-cookie%-)(.+)(/.+)$");
-        local _, tenant_uuid = tenant:match("^(vpaas%-magic%-cookie%-)(.+)$")
-        if tenant_uuid_from_kid ~= tenant_uuid then
+        if kid_parse_cache_obj.tenant ~= tenant then
```

Comment:
I think this flow doesn't always parse `kid_parse_cache_obj.tenant`?

Reply:
I dropped the vpaas_tenants_cache and also fix that kid_parse_cache_obj.tenant is always parsed.